packages/config-plugins/src/Plugin.types.ts

```diff
@@ -117,6 +117,10 @@ export interface ModConfig {
      * Modify the `android/app/src/main/<package>/MainActivity.java` as a string.
      */
     mainActivity?: Mod<AndroidPaths.ApplicationProjectFile>;
+    /**
+     * Modify the `android/app/src/main/<package>/MainApplication.java` as a string.
+     */
+    mainApplication?: Mod<AndroidPaths.ApplicationProjectFile>;
```

Comment:
Update the docs https://github.com/expo/expo/blob/master/docs/pages/guides/config-plugins.md